### DISCUSSION TITLE
How to call VSC command from vimrc?

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [VSCodeVim/Vim#9346] [Q&A]
I'm trying to figure out how to bind `<leader>sv` to split editor in VSC. I can't find any documentation about `vimrc` integration with VSC. Any idea how to do that?

**Comment by nines75** [comment]
```vim
nnoremap <leader>sv workbench.action.splitEditor
```

Note that you need to reload VSCode after updating your vimrc for it to take effect.

Reference: https://github.com/VSCodeVim/Vim/blob/a573301/src/configuration/vimrcKeyRemappingBuilder.ts#L159
